### PR TITLE
Wfprev 46

### PR DESCRIPTION
**Pull request by yzlucas**

### CODE REVIEW

**Comment by yzlucas** [comment]
@vivid-cpreston  Shall I reference the build-full-environment.yml from WFHR for the deploy.yml here?

File: terraform/alb.tf [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    }
+  }
+}
+
+
+//////////////////////////////
+/// TARGET GROUP RESOURCES ///
+//////////////////////////////
+resource "aws_alb_target_group" "wfprev_api" {
+  name                 = "wfprev-payroll-api-${var.TARGET_ENV}"
```
Comment (ssylver93) [inline terraform/alb.tf:60]: Should this be changed from wfprev-payroll-api to wfprev-api?

**Review by ssylver93 — CHANGES_REQUESTED** [review]
One comment about wfprev-payroll-api naming

**Comment by yzlucas** [comment]
@sukhpalp @ssylver93  
Thanks, cleaned up after our touch-base

**Pull request merged**